alliance/manage.py

```diff
@@ -3,6 +3,6 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "alliance.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "alliance.config.settings.base")
```

Comment:
I thought that first syntax made "alliance.config.settings.base" the default in case the env variable was not set. Maybe I misunderstand...?